pkg/apis/sparkoperator.k8s.io/v1beta2/types.go

```diff
@@ -187,6 +187,9 @@ type SparkApplicationSpec struct {
 	// Mode is the deployment mode of the Spark application.
 	// +kubebuilder:validation:Enum={cluster,client}
 	Mode DeployMode `json:"mode,omitempty"`
+	// Pass the proxyUser
```

Comment:
Can you elaborate more in the comment for this field? Also please follow Go's comment convention like `ProxyUser specifies the proxy user to impersonate when submitting the application. It maps to the command flag "--proxy-user" in spark-submit.`.

Reply:
Ok, will do.